packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -468,21 +477,23 @@ const EVENTS = {
     event: 'account_recovery_option_print',
   },
   // Save hint page
-  'flow.settings.account-recovery.hint-submit': {
+  // A user on the "hint" step of the account recovery flow clicks the submit button to save the hint they entered into the text input
+  'flow.settings.account-recovery.create-hint.submit': {
     group: GROUPS.settings,
-    event: 'account_recovery_hint_submit',
+    event: 'account_recovery_create_hint_submit',
   },
-  'flow.settings.account-recovery.hint-success': {
+  'flow.settings.account-recovery.create-hint.success': {
     group: GROUPS.settings,
-    event: 'account_recovery_hint_success',
+    event: 'account_recovery_create_hint_success',
   },
-  'flow.settings.account-recovery.hint-fail': {
+  'flow.settings.account-recovery.create-hint.fail': {
     group: GROUPS.settings,
-    event: 'account_recovery_hint_fail',
+    event: 'account_recovery_create_hint_fail',
   },
-  'flow.settings.account-recovery.hint-skip': {
+  // A user exits hint step of the account recovery key flow without saving a hint, either by exiting the flow entirely or by skipping to the next step
```

Comment:
I'm a big fan of the verbose comments ❤️ This one could probably be simplified though to `A user exits hint step of the account recovery key flow without saving a hint` because there isn't really a "next step" in the flow